### PR TITLE
add codenewbie.org

### PR DESCRIPTION
**Pull request by jchyng**
add codenewbie.org

### CODE REVIEW

**Comment by RyuaNerin** [comment]
Hello. Could you comment an example of a source site and a copy site?
I could be wrong, but it looks like ordinary to me.

**Comment by jchyng** [comment]
I'm sorry. It was a mistake 
So I corrected myself

**Comment by sh-cho** [comment]
stackexchange.com is not a fake site.

File: hosts.txt [hunk 1]
```diff
@@ -161,3 +161,4 @@ www.faqcode4u.com
 xspdf.com
 yingqusp.com
 202psj.tistory.com
+stackexchange.com
```
Comment (sh-cho) [inline hosts.txt:164]: stackexchange.com is not a fake site.

**Review by sh-cho — CHANGES_REQUESTED** [review]

**Pull request closed without merging**